openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/VehicleFlagEncoder.java

```diff
@@ -320,13 +320,6 @@ protected double getSpeed(ReaderWay way) {
             }
         }
 
-        if (way.hasTag(KEY_ACCESS)) // Runge  //https://www.openstreetmap.org/way/132312559
```

Comment:
Removed this code because it effectively duplicates the logic behind by `destinationSpeed`.